rust/osm2lanes/src/tag/mod.rs

```diff
@@ -90,6 +91,23 @@ impl Tags {
         self.map.get(k.as_ref()).map(String::as_str)
     }
 
+    /// Get the value for the given key and parses it into T.
+    pub fn _get_and_parse<V: FromStr>(&self, key: &str) -> Option<V> {
```

Comment:
a `pub` function with an underscore prefix?
I think both can be removed and then it will be consistent.

Reply:
(you could use this function int he implementation of the below, otherwise I may suggest just removing it for now, as I don't know when we would use it over the below.